src/V3Config.cpp

```diff
@@ -336,6 +336,7 @@ class V3ConfigFile final {
         }
     }
     bool waive(V3ErrorCode code, const string& match) {
+        if (code < V3ErrorCode::EC_FIRST_WARN || code.pretendError()) return false;
```

Comment:
It is ok to turn off pretend errors. Let's also cleanup...

```suggestion
        if (code.hardError()) return false;
```

then make in V3Error
      bool hardError() const { return m_e < V3ErrorCode::EC_FIRST_WARN; }

Please also use this function anywhere else that compare was.